analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -247,11 +247,18 @@ def perform_analysis(args, skip_handler, context, actions, metadata):
 
         pre_analyze = [a for a in actions
                        if a.analyzer_type == ClangSA.ANALYZER_NAME]
+        pre_anal_skip_handler = None
+
+        # Skip list is applied only in pre-analysis
+        # if --ctu-collect or --stats-collect  was called explicitly
+        if (ctu_collect and not ctu_analyze) or "stats_output" in args:
```

Comment:
Shouldn't we check if a value is set for stats_output as we do at other places?
https://github.com/Ericsson/codechecker/blob/9c5dc8202576fb2af98428946f7f1624842cf5a6/analyzer/codechecker_analyzer/analyzer.py#L109
If the flag is activated it will be in the args Namespace even if the user did not set it.